test/helpers/fixture_deploy_helper.rb

```diff
@@ -47,14 +47,18 @@ def deploy_raw_fixtures(set, wait: true, bindings: {})
 
   def deploy_dir_without_profiling(dir, wait: true, allow_protected_ns: false, prune: true, bindings: {}, sha: nil)
     current_sha = sha || SecureRandom.hex(6)
+    version_info = {
+      kube_client_version: KubernetesDeploy::IntegrationTest::KUBE_CLIENT_VERSION,
+      kube_server_version: KubernetesDeploy::IntegrationTest::KUBE_SERVER_VERSION
+    }
```

Comment:
This isn't used anymore; please remove the changes in this file..

Reply:
Will do.